### PR TITLE
solution

### PR DESCRIPTION
**Pull request by ivan-shpynda**

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days * 40 >= 280) {
+    return (days * 40) - 50;
+  } else if (days * 40 >= 120) {
```
Comment (VinnieJ-2k20) [inline src/calculateRentalCost.js:33]: If you write `return` inside `if` block, using `else` after that doesn't make sense.
Structure should be like this:

```
if (condition1) {
  return x;
}

if (condition2) {
  return y
}

return z;
```

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
Some edits needed.

If you have calculation that is used multiple times (like `days * 40` here), move it to variable and execute only once
Reusable data, like daily cost or discounts should also be used with variables.

File: src/calculateRentalCost.js (hunk 1 above)
Comment (ivan-shpynda) [reply src/calculateRentalCost.js]: thank you for your remarks. I've fixed the code

**Review by VinnieJ-2k20 — APPROVED** [review]
`days * price` is calculated three times. It also should be moved to a variable.
For now, i'll approve.
Remember about it for future tasks.

**Pull request closed without merging**